litmus-portal/frontend/src/components/CustomResultModal/styles.ts

```diff
@@ -0,0 +1,87 @@
+import { makeStyles } from "@material-ui/core";
+
+export const useStyles = makeStyles((theme) => ({
+	testHeading: {
+		marginTop: theme.spacing(6.25),
+		fontFamily: "Ubuntu",
+		fontSize: 25,
+	},
+	testType: {
+		fontSize: 17,
+		fontFamily: "Ubuntu",
+		paddingRight: theme.spacing(1.25),
+	},
+	testResult: {
+		color: "#109B67",
+		fontSize: 17,
+		fontFamily: "Ubuntu",
+		paddingLeft: theme.spacing(1.25),
+	},
+	modal: {
+		display: "flex",
+		alignItems: "center",
+		justifyContent: "center",
+	},
+	paper: {
+		minWidth: 981,
+		minHeight: 700,
+		backgroundColor: theme.palette.background.paper,
+		border: "2px solid #000",
```

Comment:
Either use borderColor property separately or use template literal and take the color from theme data like so: theme.palatte.common.black (if you want just black that is). This might seem extra work but It'll help in customizability later.

Reply:
- [x] Fixed